fontforge/plugin.c

```diff
@@ -370,7 +372,7 @@ static bool DiscoverPlugins(int do_import) {
             Py_DECREF(tmp);
             if (PyIter_Check(tmp2)) {
                 while ((str = PyIter_Next(tmp2))) {
-                    tmp = PyUnicode_AsASCIIString(str);
+                    tmp = PyUnicode_AsUTF8String(str);
```

Comment:
You should be able to use `PyUnicode_AsUTF8` and skip having to manage temporary python objects/freeing anything yourself

Reply:
Done